test/e2e/instrumentation/monitoring/custom_metrics_deployments.go

```diff
@@ -210,9 +229,18 @@ func prometheusExporterPodSpec(metricName string, metricValue int64, port int32)
 	}
 }
 
-// CreateAdapter creates Custom Metrics - Stackdriver adapter.
-func CreateAdapter() error {
-	stat, err := framework.RunKubectl("create", "-f", "https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/adapter-beta.yaml")
+// CreateAdapterForOldResourceModel creates Custom Metrics - Stackdriver adapter set up to work with
+// old Stackdriver resource model.
+func CreateAdapterForOldResourceModel() error {
```

Comment:
Maybe we could add CreateAdapter() that just calls CreateAdapterForOldResourceModel()? That way autoscaling tests don't need to know about new/old resource models and we can later swap them easily. WDYT @kawych?

Reply:
Done.